checker/tests/src/tests/IGJTest.java

```diff
@@ -1,33 +1,45 @@
 package tests;
 
+import static org.checkerframework.framework.test.TestConfigurationBuilder.buildDefaultConfiguration;
+import static org.checkerframework.framework.test.TestConfigurationBuilder.getDefaultConfigurationBuilder;
 import static org.junit.Assert.assertFalse;
 
 import java.io.File;
 import java.io.FileWriter;
 import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
 import java.util.Scanner;
 
-import org.checkerframework.framework.test.CheckerTest;
+import org.checkerframework.framework.test.TestConfiguration;
+import org.checkerframework.framework.test.TestConfigurationBuilder;
+import org.checkerframework.framework.test.TestUtilities;
+import org.checkerframework.framework.test.TypecheckExecutor;
+import org.checkerframework.framework.test.TypecheckResult;
+
 import org.junit.Ignore;
 import org.junit.Test;
+import org.junit.runners.Parameterized.Parameters;
 
+//TODO: The IGJ has a completely different testing mechanism from the rest of the framework
+//TODO: We should probably standardize
 // Also see file FailedTests, that contains currently-failing tests.
 /**
  * JUnit tests for the IGJ annotation checker.
  */
-public class IGJTest extends CheckerTest {
+public class IGJTest {
```

Comment:
The IGJ Checker was the only test that didn't fit the usual mold.  It required a few of the old methods from the test framework which have been moved here.  It should be updated to match our testing conventions.